create-env

```diff
@@ -231,6 +230,21 @@ if [ -e \$config_path ]; then
    export XENON_CONFIG=\$config_path
 fi
 
+# grab a local cutax installation if we have one
+MIDWAY_CUTAX_DIR=/dali/lgrandi/xenonnt/software/cutax/v${cutax_version}
+OSG_CUTAX_DIR=/xenon/software/cutax/v${cutax_version}
+if [ "x\${CUTAX_LOCATION}" = "x" ]; then
+  for dir in ${MIDWAY_CUTAX_DIR} ${OSG_CUTAX_DIR}; do
+    if [ -e \$dir ]; then
+       export CUTAX_LOCATION=\$dir
+    fi
+  done
+fi
```

Comment:
My bash is not my best, maybe you already do this, but do we want to raise some error if this dir does not exist (and are not running github actions)

Reply:
This is just an if statement checking if those paths exist, so no error raising necessary (you can see the build already succeeded in actions)

Reply:
Yeah, good point. 

It was more a suggestion if we wanted this as it can lead to confusion after building right if something is not available. Maybe not a super probable situation though.